### PR TITLE
Add Skipper stream force update

### LINKED ISSUE #2412: Add support for force stream update
With Skipper introducing https://github.com/spring-cloud/spring-cloud-skipper/issues/714 we need to have stream update operation with the `force` operation.

### LINKED ISSUE #2414: Upgrade Skipper to 1.1.0 snapshot for SCDF 1.7.x

### PR DESCRIPTION
**Pull request by ilayaperumalg**
- Use skipper `force` update functionality
 - Set upgrade request with `force` and `appNames` options
 - Update test

Resolves #2412
Resolves #2414

### CODE REVIEW

File: spring-cloud-dataflow-rest-resource/src/main/java/org/springframework/cloud/dataflow/rest/UpdateStreamRequest.java [hunk 1]
```diff
@@ -31,15 +33,26 @@
 
 	private Map<String, String> updateProperties;
 
+	private boolean force;
+
+	private String[] appNames;
```
Comment (markpollack) [inline spring-cloud-dataflow-rest-resource/src/main/java/org/springframework/cloud/dataflow/rest/UpdateStreamRequest.java]: Switch to List instead?

File: spring-cloud-dataflow-skipper-docs/pom.xml [hunk 2]
```diff
@@ -39,6 +39,12 @@
 			<artifactId>json-path</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
+			<groupId>io.projectreactor</groupId>
+			<artifactId>reactor-core</artifactId>
+			<version>3.1.8.RELEASE</version>
+			<scope>test</scope>
+		</dependency>
```
Comment (markpollack) [inline spring-cloud-dataflow-skipper-docs/pom.xml]: why is this here?

**Comment by markpollack** [comment]
rebase, changed String[] to List<String>

merged as 8cdc072373368a52c5d78bf0c6eae02da983916c

File: spring-cloud-dataflow-skipper-docs/pom.xml (hunk 2 above)
Comment (ilayaperumalg) [reply spring-cloud-dataflow-skipper-docs/pom.xml]: This was needed to have Skipper related doc tests using the specific reactor version that is used by Skipper 1.1.0.
